### DISCUSSION TITLE
Questions Regarding "Digitaler-Impfnachweis"-Certificate

### DISCUSSION THREAD

**Discussion by 1fabi0** [Digitaler-Impfnachweis/certification-apis#20] [Q&A] [answered]
This is not my initial Question because after reading the web page from Ubirch regarding the "Digitaler-Impfnachweiß"(would really appreciate if you link that page in your documentation so there is a easy entry point) but I still have some security relevant questions regarding the certificates.

Do Certificates have a expiration Date, or is it possible to mark that a newer one is available?
What happens when certificates should be replaced e.g. because of new vaccination? Is the old one marked as expired?
Can other Vaccinations, not against Covid-19, be also safed on the "Digitaler-Impfnachweiß" and used on e.g. the Airport? Can't find anything explicit saying that it's possible or not.
What happens if cryptography that is used is cracked? Is it possible to fast sign new certificates for all users/ for the users that are affected? (even let's encrypt is prepared for this so why not our vaccination Certificates?)

**Comment by oliver-steinbrecher** [comment, marked as answer]
Please have a look here https://digitaler-impfnachweis-app.de/

**Reply by 1fabi0** [reply]
I am sorry to tell you but I can't find any answer to any of my questions I just now have one more question: what happens if a doctor fakes these certificates for it's pacients e.g. He is against vaccination can cerzificates be marked as invalid certificates in the latest block?

**Reply by LeftyMay** [reply]
and what happens if the Earth stop spinning.... :-)

**Reply by 1fabi0** [reply]
I think this question is less interesting then the more probable cases that a doctor fakes these certificates for the Querdenker community or that someone also wants to have other vaccinations on it's Nachweis... Love this English-German mix

**Reply by rumpelrausch** [reply]
...or that people just share their qrcode. I doubt that every club or restaurant will check the data against the person's ID or passport. You just hold your fon against some camera and that's it.
The paper variant seems much safer to me since it involves some thinking on the other side. The cert should be easy to present but hard to verify, not vice versa. Then whoever verifies it is responsible for the outcome. You let people in with fake certs even though could could have checked their ID? Same legal issue as faking it.
